### ISSUE TITLE
i think ur mistaken

### ISSUE THREAD

**Issue by Phant0m-a** [guchengxi1994/guchengxi1994#1]
Elder ring is clearly the winner here ;)

**Comment by guchengxi1994** [comment]
@Phant0m-a  emmm... Bloodborne is my first PS4 game, and I think Cthulhu Mythos is much more attractive to me.😂

**Comment by Phant0m-a** [comment]
🤐 Bloodborne ... is a horror game and so is Cthulhu Mythos ... gives me Slander men type kinda vibes...

**Comment by guchengxi1994** [comment]
yes, Bloodborne is scary, especially the musics. There are a lot of  screams and whispers in this game and most monsters are like `Godrick the Grafted`, but guns and actions makes this game quite different from Dark souls and Elder ring

**Comment by Phant0m-a** [comment]
Intresting, feels lot like Devil may cry!!! don't you think. Witchers also closely ressembles to this...
tbh i find it hard to play dark games...maybe cuz of its graphics...

**Comment by guchengxi1994** [comment]
I have to say, devil may cry is a little difficult for me, especially 3 and 5

**Comment by Phant0m-a** [comment]
😅Have you tried slender men... i did back in high school... it was the most horrifying game for me!!!
those were some crazy years!

**Comment by guchengxi1994** [comment]
No... I seldom play a horror game or watch a horror film since I played a game called `Dread Out` in university. In this game you will play a girl student, and fight with monsters( or ghosts) with a mobile phone... I think this kind of game is stupid and `Dread Out` really scared me...

**Comment by Phant0m-a** [comment]
`Dread out` ... oh yeah it's kinda stupid. Try `FarCry ` from 3 to 6. amazing storyline...
Or wanna play any multiplayer game with me!!

**Comment by guchengxi1994** [comment]
do you have a steam account? maybe we can play elder ring DLCs on the steam later.

**Comment by Phant0m-a** [comment]
😐 sorry, i kinda don't have elder rings let alone its Dlc's. yeah it sucks

**Comment by Phant0m-a** [comment]
hey man you here? do u have any social media?

**Comment by guchengxi1994** [comment]
I have steam,epic,  wechat and qq accounts

**Comment by Phant0m-a** [comment]
Do u have facebook or WhatsApp....so I could like contact you there...

**Comment by guchengxi1994** [comment]
you can search guchengxi1994@qq.com on facebook, but i log on facebook infrequently

**Comment by Phant0m-a** [comment]
Ha ur hard to find there😅

**Comment by guchengxi1994** [comment]
maybe search `xiaoshuyui` instead?

**Comment by Phant0m-a** [comment]
https://www.facebook.com/TheAhsanA
😅 still cant find you...add me instead...here's the link

**Comment by guchengxi1994** [comment]
my facebook account has been blocked, i think it must because i  never logged in since 2019... I can find you on facebook but cannot add you as a friend. I will try it later when my account is ok

**Comment by Phant0m-a** [comment]
ah that's sad!
